remoteworkitem/trackerquery_repository.go

```diff
@@ -37,7 +41,7 @@ func (r *GormTrackerQueryRepository) Create(ctx context.Context, query string, s
 		ID:        string(tq.ID),
 		Query:     query,
 		Schedule:  schedule,
-		TrackerID: int(tid)}
+		TrackerID: string(tid)}
```

Comment:
By using string(tid) you're recreating the same issue as https://github.com/almighty/almighty-core/issues/189

See https://github.com/almighty/almighty-core/pull/197/files#diff-fa20f0fa64c649029bd4d324888830a0R129